### PR TITLE
Create k8s updater blog

### PR DESCRIPTION
**Pull request by tashimi**
tenor-149491620

Signed-off-by: Tasha Drew <tasha.drew@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Last week, Habitat core team member [Elliott Davis](https://twitter.com/libsysguy) demonstrated the next step in the Habitat to Kubernetes workflow: Habitat application rebuilds (which can be triggered by source code and upstream dependency updates) can now be automatically deployed to applications running on Kubernetes. (He will also be demonstrating [this live at Kubecon EU, May 2nd at 2pm](http://sched.co/Dqua) with the one and only [Fletcher Nichol](https://twitter.com/fnichol) for those who are in town to check out and ask questions).
```
Comment (cnunciato) [inline www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: `to check it out and ask`

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Last week, Habitat core team member [Elliott Davis](https://twitter.com/libsysguy) demonstrated the next step in the Habitat to Kubernetes workflow: Habitat application rebuilds (which can be triggered by source code and upstream dependency updates) can now be automatically deployed to applications running on Kubernetes. (He will also be demonstrating [this live at Kubecon EU, May 2nd at 2pm](http://sched.co/Dqua) with the one and only [Fletcher Nichol](https://twitter.com/fnichol) for those who are in town to check out and ask questions). 
+
+Every Habitat application and package has channels that running services can subscribe to for update. The defaults are `unstable` and `stable`, but a user can define as many as they would like to, for example `development`, `qa`, `staging` and `production`. You can now subscribe your Habitat-built applications running on Kubernetes to a Habitat Builder channel of your choice, and they will be able to follow standard Habitat automation and update themselves when an update is made available on their channel.
```
Comment (cnunciato) [inline www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: `for updates.`

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Last week, Habitat core team member [Elliott Davis](https://twitter.com/libsysguy) demonstrated the next step in the Habitat to Kubernetes workflow: Habitat application rebuilds (which can be triggered by source code and upstream dependency updates) can now be automatically deployed to applications running on Kubernetes. (He will also be demonstrating [this live at Kubecon EU, May 2nd at 2pm](http://sched.co/Dqua) with the one and only [Fletcher Nichol](https://twitter.com/fnichol) for those who are in town to check out and ask questions). 
+
+Every Habitat application and package has channels that running services can subscribe to for update. The defaults are `unstable` and `stable`, but a user can define as many as they would like to, for example `development`, `qa`, `staging` and `production`. You can now subscribe your Habitat-built applications running on Kubernetes to a Habitat Builder channel of your choice, and they will be able to follow standard Habitat automation and update themselves when an update is made available on their channel. 
+
+Check out Elliott’s demo of the Habitat Updater Service on Youtube, and let us know what you think!
```
Comment (cnunciato) [inline www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: `YouTube`

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+![Image of Kubernetes and Hab](https://www.habitat.sh/images/infographics/deploy-services-to-kubernetes-with-habitat-flow-6ddca9cd.png)
+
+### Got questions? 
+* [Ask and answer questions on the Habitat forums](https://forums.habitat.sh/) 
+* [Chat with the Habitat Community on Slack](http://slack.habitat.sh/) 
+* [Learn more about Habitat](https://www.habitat.sh/) 
+
+### Read more: 
+* [Habitat Operator for Kubernetes on GitHub](https://github.com/habitat-sh/habitat-operator) 
+* [Habitat + Open Service Broker]( link )
```
Comment (cnunciato) [inline www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: Looks like this one still needs a link: https://www.habitat.sh/blog/2018/05/Hab-OSB/

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Once we’ve exported your application to the format you want, the application should be able to perform within that ecosystem in a highly performant and native way. This requires additional automation and integration in order to ensure application behavior knows how to behave natively and intelligently. For the Cloud Native world, specifically Kubernetes, the Habitat team has been leveraging a bunch of exciting technology to ensure this experience, with exporters to Helm and Kubernetes manifests, a Habitat Operator to ensure operational behavior and allow users to leverage powerful native Kubernetes capabilities while keeping their applications in sync, and an Open Service Broker.
```
[the remark this answers is not in the document]
Comment (tashimi) [reply www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: yup

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md (hunk 1 above)
Comment (tashimi) [reply www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: yup

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md (hunk 2 above)
Comment (tashimi) [reply www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: yup

File: www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md (hunk 3 above)
Comment (tashimi) [reply www/source/blog/2018-05-01-Auto-App-Updates-k8s.html.md]: yup

**Pull request merged**